app/src/main/java/com/globant/example/mentorapp/subscriberDetails/domain/model/ListReposEntity.java

```diff
@@ -0,0 +1,411 @@
+    @SerializedName("subscribers_url")
+    private String subscribersUrl;
+    @SerializedName("subscription_url")
+    private String subscriptionUrl;
+    @SerializedName("commits_url")
+    private String commitsUrl;
+    @SerializedName("git_commits_url")
+    private String gitCommitsUrl;
+    @SerializedName("comments_url")
+    private String commentsUrl;
+    @SerializedName("issue_comment_url")
+    private String issueCommentUrl;
+    @SerializedName("contents_url")
+    private String contentsUrl;
+    @SerializedName("compare_url")
+    private String compareUrl;
+    @SerializedName("merges_url")
+    private String mergesUrl;
+    @SerializedName("archive_url")
+    private String archiveUrl;
+    @SerializedName("downloads_url")
+    private String downloadsUrl;
+    @SerializedName("issues_url")
+    private String issuesUrl;
+    @SerializedName("pulls_url")
+    private String pullsUrl;
+    @SerializedName("milestones_url")
+    private String milestonesUrl;
+    @SerializedName("notifications_url")
+    private String notificationsUrl;
+    @SerializedName("labels_url")
+    private String labelsUrl;
+    @SerializedName("releases_url")
+    private String releasesUrl;
+    @SerializedName("deployments_url")
+    private String deploymentsUrl;
+    @SerializedName("created_at")
+    private String createdAt;
+    @SerializedName("updated_at")
+    private String updatedAt;
+    @SerializedName("pushed_at")
+    private String pushedAt;
+    @SerializedName("git_url")
+    private String gitUrl;
+    @SerializedName("ssh_url")
+    private String sshUrl;
+    @SerializedName("clone_url")
+    private String cloneUrl;
+    @SerializedName("svn_url")
+    private String svnUrl;
+    private String homepage;
+    private Integer size;
+    @SerializedName("stargazers_count")
+    private Integer stargazersCount;
+    @SerializedName("watchers_count")
+    private Integer watchersCount;
+    private String language;
+    @SerializedName("has_issues")
+    private Boolean hasIssues;
+    @SerializedName("has_projects")
+    private Boolean hasProjects;
+    @SerializedName("has_downloads")
+    private Boolean hasDownloads;
+    @SerializedName("has_wiki")
+    private Boolean hasWiki;
+    @SerializedName("has_pages")
+    private Boolean hasPages;
+    @SerializedName("forks_count")
+    private Integer forksCount;
+    @SerializedName("mirror_url")
+    private Object mirrorUrl;
+    @SerializedName("open_issues_count")
+    private Integer openIssuesCount;
+    @SerializedName("forks")
+    private Integer forks;
+    @SerializedName("open_issues")
+    private Integer openIssues;
+    private Integer watchers;
+    @SerializedName("default_branch")
+    private String defaultBranch;
+
+    public Integer getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getFullName() {
+        return fullName;
+    }
+
+    public Boolean get_private() {
```

Comment:
Respect java conventions. Avoid "_" in methods name. The use of "_" in methods names is for unit tests mainly.